README.md

```diff
@@ -31,3 +31,30 @@ The repository has
 as a submodule, and testing and builds are set-up to use the submodule version.
 When changes are made in `concordium-contracts-common` they should be propagated
 to this repository.
+
+## Contributing
+
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg)](https://github.com/Concordium/.github/blob/main/.github/CODE_OF_CONDUCT.md)
+
+This repository's CI automatically checks formatting and common problems in rust.
+Changes to any of the packages must be such that
+
+- ```cargo clippy --all``` produces no warnings
+- ```rustfmt``` makes no changes.
+
+Everything in this repository should build with stable rust at the moment (at least version 1.44 and up), however the fmt tool must be from a nightly release since some of the configuration options are not stable. One way to run the `fmt` tool is
```

Comment:
```suggestion
Everything in this repository should build with rust version 1.53 (it should work with 1.51 and up, but we do not test regularly) however the `fmt` tool must be from a nightly release since some of the configuration options are not stable. One way to run the `fmt` tool is
```